docs/dietpi_docs_todo.md

```diff
@@ -50,6 +50,36 @@ Open point, has to be cleared.
 
 - [x] Petru to enable new file structure
 
+### 2.3 Content order
+
+**Proposal:** The content of the HOME area is reordered by exchanging Supported SBCs with Optimised software list and Advanced configuration with Community tutorials.  
+This gives the following benefits:
+
+- "Advanced configuration" is nearer to the system usage description
+- All links to DietPi.com forum ("Community tutorials" and "Troubleshooting" are consecutive)
+
+Then the main order will be:
+
+1. Overview
+2. Installation
+3. Getting started
+4. Supported SBCs
+5. Advanced configuration
+6. Community tutorials
+7. Troubleshooting
+8. ...
+
+#### Decision?
```

Comment:
I do nearly fully agree. Somehow the "Supported SBCs" does not fit well inside. The order is moreless perfectly the one that one would go through when being about to install and setup DietPi, from base steps to basic config to advanced configs, tutorials and finally troubleshooting, that is perfectly harmonic IMO. Only the "Supported SBCs" is more something that is generally interesting but not something that one requires when having the SBC in the table and wants to get started.

Maybe best is still 2. since "Overview" also has the "What is supported?" header and links to that page. Another idea would be to move it as tab to the top navigation, before or after "OPTIMISED SOFTWARE", what you think?

Reply:
Yes, I also am not sure where the section "Supported SBCs" shall be located.  
I am open to all your proposals, my preference would be an own top navigation where every board will be one navigation line when expanded. This leads to a quite quick navigation.

Reply:
Proposal: Decision in telecon.